Project/dataplot.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
Recommend  adding a block of text in the beginning to describe functionality, something like...
"""
Written By: a name
For: Winkler Lab/CSE599 Winter Quarter 2016
Purpose: a purpose

To Do:
+A couple
+of ideas
+of where to go next
"""